src/deadline_worker_agent/sessions/actions/run_attachment_upload.py

```diff
@@ -71,54 +71,55 @@ def __init__(
 
     def set_step_script(
         self,
-        manifest_paths_by_root: dict[str, list[str]],
-        out_rel_dirs_by_root: dict[str, list[str]],
         s3_settings: JobAttachmentS3Settings,
+        worker_manifest_properties_list: list,
```

Comment:
nit: can we type the expected objects in the list?
```suggestion
        worker_manifest_properties_list: list[WorkerManifestProperties],
```